content/docs/guides/cadvisor.md

```diff
@@ -0,0 +1,116 @@
+    command:
+    - --config.file=/etc/prometheus/prometheus.yml
+    volumes:
+    - ./prometheus.yml:/etc/prometheus/prometheus.yml:ro
+    depends_on:
+    - cadvisor
+  cadvisor:
+    image: google/cadvisor:latest
+    container_name: cadvisor
+    ports:
+    - 8080:8080
+    volumes:
+    - /:/rootfs:ro
+    - /var/run:/var/run:rw
+    - /sys:/sys:ro
+    - /var/lib/docker/:/var/lib/docker:ro
+    depends_on:
+    - redis
+  redis:
+    image: redis:latest
+    container_name: redis
+    ports:
+    - 6379:6379
+```
+
+This configuration instructs Docker Compose to run three services, each of which corresponds to a [Docker](https://docker.com) container:
+
+1. The `prometheus` service uses the local `prometheus.yml` configuration file (imported into the container by the `volumes` parameter).
+1. The `cadvisor` service exposes port 8080 (the default port for cAdvisor metrics) and relies on a variety of local volumes (`/`, `/var/run`, etc.).
+1. The `redis` service is a standard Redis server. cAdvisor will gather container metrics from this container automatically, i.e. without any further configuration.
+
+To run the installation:
+
+```bash
+docker-compose up
+```
+
+If Docker Compose successfully starts up all three containers, you should see output like this:
+
+```
+prometheus  | level=info ts=2018-07-12T22:02:40.5195272Z caller=main.go:500 msg="Server is ready to receive web requests."
+```
+
+You can verify that all three containers are running using the [`ps`](https://docs.docker.com/compose/reference/ps/) command:
+
+```bash
+docker-compose ps
+```
+
+Your output will look something like this:
+
+```
+   Name                 Command               State           Ports
+----------------------------------------------------------------------------
+cadvisor     /usr/bin/cadvisor -logtostderr   Up      8080/tcp
+prometheus   /bin/prometheus --config.f ...   Up      0.0.0.0:9090->9090/tcp
+redis        docker-entrypoint.sh redis ...   Up      0.0.0.0:6379->6379/tcp
+```
+
+## Exploring the cAdvisor web UI
+
+You can access the cAdvisor [web UI](https://github.com/google/cadvisor/blob/master/docs/web.md) at `http://localhost:8080`. You can explore stats and graphs for specific Docker containers in our installation at `http://localhost:8080/docker/<container>`. Metrics for the Redis container, for example, can be accessed at `http://localhost:8080/docker/redis`, Prometheus at `http://localhost:8080/docker/prometheus`, and so on.
+
+## Exploring metrics in the expression browser
+
+cAdvisor's web UI is a useful interface for exploring the kinds of things that cAdvisor monitors, but it doesn't provide an interface for exploring container *metrics*. For that we'll need the Prometheus [expression browser](/docs/visualization/browser), which is available at `http://localhost:9090/graph`. You can enter Prometheus expressions into the expression bar, which looks like this:
+
+![Prometheus expression bar](/assets/prometheus-expression-bar.png)
```

Comment:
Did you mean to include the missing image in this PR already?

Reply:
Ah ok, was confused because the preview at https://github.com/lucperkins/docs/blob/392eaa667e2eed54e89fce9907c76559286f8668/content/docs/guides/cadvisor.md#prometheus-configuration didn't show it, but that makes sense in the context of GitHub.